assemblies/assembly-enabling-authentication.adoc

```diff
@@ -1,39 +1,46 @@
 [id='enabling-authentication']
 = Enabling authentication in {product}
 
-Authentication within {product} facilitates user sign-in, identification, and access to external resources.
-It supports multiple authentication providers.
-
-Authentication providers are typically used in the following ways:
-
-- One provider for sign-in and identification.
-- Additional providers for accessing external resources.
-
-The {product} supports the following authentication providers:
-
-//Auth0:: `auth0`
-//Atlassian:: `atlassian`
-Microsoft Azure::
-`microsoft`
-//Microsoft Azure Easy Auth:: `azure-easyauth`
-//Bitbucket:: `bitbucket`
-//Bitbucket Server:: `bitbucketServer`
-//Cloudflare Access:: `cfaccess`
-GitHub:: `github`
-//GitLab:: `gitlab`
-//Google:: `google`
-//Google IAP:: `gcp-iap`
-Keycloak:: `oidc`
-//Okta:: `okta`
-//OAuth 2 Custom Proxy:: `oauth2Proxy`
-//OneLogin:: `onelogin`
-//SAML:: `saml`
-
-For each provider that you want to use, follow the dedicated procedure to complete the following tasks:
+Your organization certainly requires you to identify and authorize users to give them access to resources, such as {product}.
+
+In {product-short}, authentication and authorization are two separate processes:
+
+. Authentication is defining the user identity, and passing on this information to {product-short}.
+Read following chapters to progressively configure authentication in {product-short}.
+
+. Authorization is defining what actions an authenticated identity can perform in {product-short}.
+See link:{authorization-book-url}[{authorization-book-title}].
+
+To start exploring {product-short} features, do not worry about authentication and authorization: after the default installation, you can authenticate with the Guest user, and use all the application features.
+
+The authentication system in {product-short} is handled by external authentication providers.
+
+{product-short} supports following authentication providers:
+
+* Red Hat Single-Sign On (RHSSO)
+* GitHub
+* Microsoft Azure
+
+To identify users in {product-short}, configure one authentication provider for sign-in and identification:
 
 . Set up the shared secret that the authentication provider and {product} require to communicate.
+
 . Configure {product} to use the authentication provider.
 
+Optionally, to add more information to the user identity, or enable access to additional external resources, configure additional providers for identification.
+
+You can use only one provider for sign-in.
```

Comment:
A user may find this a little confusing, we're talking about adding additional resources (what are these?) and configuring additional providers (I presume RHSSO, Github and Azure) but then correctly telling them to only use one provider. 

Maybe it could be clearer that they can configure multiple providers but only use one of these providers to sign-in?